spreadsheet-reader.patch

```diff
@@ -16,13 +16,13 @@ index 75351b7..88c5ef4 100644
 --- a/php-excel-reader/excel_reader2.php
 +++ b/php-excel-reader/excel_reader2.php
 @@ -675,7 +675,9 @@ class Spreadsheet_Excel_Reader {
- 	 */
- 	var $dateFormats = array (
- 		0xe => "m/d/Y",
--		0xf => "M-d-Y",
+ 	 */

```

Comment:
We need to be very careful with this file. The mac newlines in this file are very important for the patch to apply as there were mac newlines in the original `spreadsheet-reader` module. Thus, I can't merge this PR until these changes are removed.

This github config `git config --global core.autocrlf true` is what causes the mac line ending to be switched automatically. We need to turn it off for this module but ensure that our editors use linux line endings.